app/controllers/admin/grove_monitor_controller.rb

```diff
@@ -15,6 +15,17 @@ def show
     authorize(:grove_monitor, :show?)
   end
 
+  def update
+    student = Student.find(params[:id])
+    if student.at_school
+      student.update_attribute(:at_school, false)
+    else
```

Comment:
while this will work, best to use update because `update_attribute` does not check validations.